src/dashboard.ts

```diff
@@ -61,6 +61,10 @@ export type ServiceOptions = {
    * The meeting action.
    */
   meetingAction?: MeetingAction;
+  /**
+   * the UI elements to be hidden.
+   */
+  hiddenElements?: number[];
```

Comment:
discussion, as an API - do we want an enum for the possible values?

Reply:
We can direct them to enable the UI Map on facetalk via feature flag dev tools to see which elements gets hidden. Imo enum would just be listing 50+ internal flag names that doesn't fully describe or show what will be hidden.
![image](https://github.com/iotum/callbridge-js/assets/64922037/0eb286a5-6969-4294-9207-f5e1312cdeae)

Reply:
are you saying the same 'hidden elements' param is also intended for the meeting (call) widget?

Reply:
No, atm its only for the dashboard. Apparently the InCallPage has a different provider for the hidden elements.